packages/next/taskfile.js

```diff
@@ -2349,9 +2349,6 @@ export async function next_compile(task, opts) {
       'shared_re_exported_esm',
       'server_wasm',
       'experimental_testmode',
-      // we compile this each time so that fresh runtime data is pulled
-      // before each publish
-      'ncc_amp_optimizer',
```

Comment:
Is the bit about pulling fresh runtime data no longer needed/true? Why do we need this change anyway?

Reply:
Expanded on the consistency argument: 
> That way we can rely on `src/compiled/**/package.json` to indicate a vendored package.

Reply:
Okay, I believe that the original reason was that this is special-cased is that `@ampproject/toolbox-optimizer` [used to download stuff in postinstall](https://github.com/ampproject/amp-toolbox/blob/4fcd0031491294e2fffb7ba4bdf2e77ec1e89b66/packages/optimizer/package.json#L12). and we wanted to avoid checking in its source in order to get the whole thing fresh each time, i guess? (which TBH is kinda bad, because it makes the build non-deterministic...).

HOWEVER they removed that behavior in 2021: https://github.com/ampproject/amp-toolbox/pull/1181. and just to double-check, the version that we've got installed [no longer has a `postinstall` script at all](https://github.com/ampproject/amp-toolbox/blob/v2.8.3/packages/optimizer/package.json). so it should be safe to do this change!

Reply:
Yeah I think it's good to change this now as this project has fizzled out and shouldn't be updated anymore anyways.

Reply:
FWIW they did release a bunch of updates (2.8.3 -> 2.10.1), e.g. some stuff including a fallback version of [that data that used to be downloaded in postinstall] with the package. so might be useful to update it anyway